stdlib/public/core/FloatingPointTypes.swift.gyb

```diff
@@ -1582,11 +1582,25 @@ extension ${Self} {
 
 extension ${Self} {
   @_transparent
+  // SWIFT_ENABLE_TENSORFLOW
+  @differentiable(adjoint: _adjointNegate)
   public static prefix func - (x: ${Self}) -> ${Self} {
     return ${Self}(Builtin.fneg_FPIEEE${bits}(x._value))
   }
 }
 
+// SWIFT_ENABLE_TENSORFLOW
+extension ${Self} {
+  @inlinable
```

Comment:
I don’t think inlinable is necessary.

Reply:
I get a SILVerifier error about referencing a non-ABI-public function from a fragile function if I remove inlinable. (`simple_math.swift` triggers it)

I think what's happening is that _transparent doesn't make the function ABI public, and mandatory inlining isn't inlining the function because we're partially applying it.

Reply:
Gotcha. If you just need it to be ABI-public without needing it serializable, just use `@usableFromInline`.